pom.xml

```diff
@@ -782,6 +782,12 @@
         <version>${version.catch-exception}</version>
         <scope>test</scope>
       </dependency>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-log4j12</artifactId>
```

Comment:
Any particular reason to use log4j and not slf4j-simple which is used elsewhere?

Reply:
Yes, the log4j is needed by the OpenShift client library and without this dependency the tests were failing with ClassNotFoundError. Btw. The scope for it is only test.